server/cli/src/graphql/auth.rs

```diff
@@ -0,0 +1,16 @@
+pub(super) const AUTH_QUERY: &str = r#"
+query AuthToken($username: String!, $password: String) {
+  root: authToken(password: $password, username: $username) {
```

Comment:
this root relates to the `GraphQlResponse` struct and `Root` struct, it seems cleaner to use struct with auto derived de/se and map graphql field to root, rather then manually accessing serde_json::Value type